### PR TITLE
Fixed bug where `getBlockInfo` throws error on genesis blocks.

### PR DESCRIPTION
**Pull request by rasmus-kirk**
## Changes

- Updated `blockInfo` so that the `bakerId` field is optional, since it will be undefined for genesis blocks.

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

**Comment by rasmus-kirk** [comment]
This is a breaking change, and will not be merged until the next major release.

**Comment by abizjak** [comment]
I am not sure about the rationale here. This is a bug where the SDK does not work. We can be academic about it and delay the release because it is a "major version bump", but ultimately the SDK in the current version is broken.

How long are we waiting with this release then?

**Comment by soerenbf** [comment]
> I am not sure about the rationale here. This is a bug where the SDK does not work. We can be academic about it and delay the release because it is a "major version bump", but ultimately the SDK in the current version is broken.
> 
> How long are we waiting with this release then?

Well we should not wait too long then. Would be nice to at least include #166 and possibly check if we can implement a good solution for #112 (I don't know if this would actually require anything breaking the current implementation or not, but we could at least investigate).

File: packages/common/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## Unreleased
+
+### Changed
```
Comment (shjortConcordium) [inline packages/common/CHANGELOG.md:5]: The section should be breaking changes. :thinking:

**Review by shjortConcordium — APPROVED** [review]

**Pull request merged**